### PR TITLE
Graph fixes

### PR DESCRIPTION
**Pull request by evulins**
Fixed the bugs listed below:

- fixed markers: should directly navigate to the calendar view (we are not showing modal for this sessions"

- click on a fixed marker > click "24 HOURS": sometimes the stats update even though the timeframe and graph do not update after clicking "24 HOURS" .  if you can't reproduce on your first try, click on a different marker and try again until it's reproduced; you shouldn't have to try more than 10 times before the bug is reproduced.

- click on a fixed marker > click "1 WEEK" > hover your mouse over one of the graph chevrons: the timeframe changes even tough no action has been taken to change the timeframe.  this bug also impacts the mobile graph, click "5 MINUTES" then hover hover the graph chevron

- hover over the graph chevron: timeframe buttons should not collapse and expand.  the below two items may be related to this bug

- fixed > click left hand side chevron: vertical line and blue dot should not be visible when clicking the chevron

- drag the TC thresholds: timeframe buttons should not collapse and expand

[Trello card](https://trello.com/c/dvX1cq18/1828-session-details-graph)

### CODE REVIEW

File: app/javascript/react/components/Graph/Graph.tsx [hunk 1]
```diff
@@ -25,22 +25,22 @@ import { updateMobileMeasurementExtremes } from "../../store/mobileStreamSlice";
...
-import { selectFixedStreamShortInfo } from "../../store/fixedStreamSelectors";
+import {
+  selectFixedExtremes,
+  selectFixedStreamShortInfo,
+} from "../../store/fixedStreamSelectors";
 import { selectMobileStreamData } from "../../store/mobileStreamSelectors";
 import { selectMobileStreamShortInfo } from "../../store/mobileStreamSelectors";
 import { useAppDispatch } from "../../store/hooks";
-import { handleLoad, handleRedraw } from "./chartEvents";
+import { handleLoad } from "./chartEvents"; // Import handleLoad
```
Comment (jankrzempek) [inline app/javascript/react/components/Graph/Graph.tsx:35]: comment nedeed?

File: app/javascript/react/components/Graph/Graph.tsx [hunk 2]
```diff
@@ -56,9 +56,12 @@ const Graph: React.FC<GraphProps> = ({ streamId, sessionType }) => {
       : selectMobileStreamShortInfo
   );
 
+  const { minMeasurementValue, maxMeasurementValue } =
+    useSelector(selectFixedExtremes);
+
   const measurements = graphData?.measurements || [];
   const unitSymbol = streamShortInfo?.unitSymbol || "";
-  const measurementType = "Particulate Matter"; // take this parameter from filters in the future
+  const measurementType = "Particulate Matter";
```
Comment (jankrzempek) [inline app/javascript/react/components/Graph/Graph.tsx:64]: Maybe let's create some special constant for such a type? I think this name is important, is it?

File: app/javascript/react/components/Graph/chartEvents.ts [hunk 3]
```diff
@@ -2,77 +2,55 @@ import Highcharts from "highcharts/highstock";
...
-    const { min, max, dataMin, dataMax } = axis.getExtremes();
-    const range = max - min;
-    const newMin = Math.max(dataMin, min - range * 0.1);
-    const newMax = Math.max(dataMin + range, max - range * 0.1);
-    axis.setExtremes(newMin, newMax);
-    updateArrowState();
-  };
-
-  const moveRight = () => {
+  const move = (direction: 'left' | 'right') => {
```
Comment (jankrzempek) [inline app/javascript/react/components/Graph/chartEvents.ts:26]: can this direction also be a type? When we use a string here and then again below it is more error-prone. Having it and using constant could help

File: app/javascript/react/components/Map/Map.tsx [hunk 4]
```diff
@@ -35,6 +40,7 @@ const Map = () => {
 
   // Hooks
   const dispatch = useAppDispatch();
+  const navigate = useNavigate(); // Initialize useNavigate
```
Comment (jankrzempek) [inline app/javascript/react/components/Map/Map.tsx:43]: this comment is nedeed?

File: app/javascript/react/components/molecules/FixedStreamStationHeader/FixedStreamStationHeader.tsx [hunk 5]
```diff
@@ -26,7 +26,8 @@ const FixedStreamStationHeader = () => {
     endTime,
   } = useSelector(selectFixedStreamShortInfo);
 
-  const streamEndTime: string = endTime ?? lastUpdate ?? moment().format("YYYY-MM-DD");
+  const streamEndTime: string =
+    endTime ?? lastUpdate ?? moment().format("YYYY-MM-DD");
```
Comment (jankrzempek) [inline app/javascript/react/components/molecules/FixedStreamStationHeader/FixedStreamStationHeader.tsx:30]: We have as I remember on master now the DateType which include such "YYYY-MM-DD". Would you mind using it?

**Review by jankrzempek — COMMENTED** [review]
Cannot test now because of migration problem, please consider those comments and treat them as optionals

File: app/javascript/react/store/fixedStreamSlice.ts [hunk 6]
```diff
@@ -16,7 +16,7 @@ interface FixedStreamState {
   averageMeasurementValue: number | null;
   isLoading: boolean;
 }
-
+const MILLISECONDS_IN_A_DAY = 24 * 60 * 60 * 1000;
```
Comment (JoannaThelema) [inline app/javascript/react/store/fixedStreamSlice.ts:19]: I see we have a duplicate in [Graph.tsx](https://github.com/HabitatMap/AirCasting/pull/679/files#diff-7471bf256dae87ada9622e22f537149dfd03e2c54b4139c0bed600bbacfb5266)

**Review by JoannaThelema — APPROVED** [review]
LGTM 🙌

File: app/javascript/react/components/Graph/Graph.tsx (hunk 2 above)
Comment (evulins) [reply app/javascript/react/components/Graph/Graph.tsx:60]: This is how it will be until we install filters.

**Review by jankrzempek — APPROVED** [review]
A W E S O M E !

**Pull request merged**